specification/mediaservices/resource-manager/Microsoft.Media/stable/2020-05-01/examples/liveevent-delete.json

```diff
@@ -38,7 +38,7 @@
             }
           },
           "PreviewLocator": null,
-          "streamingPolicyName": null
+          "StreamingPolicyName": null
```

Comment:
case change again - Breaking or not?

Reply:
Case changes ARE breaking

Reply:
This is in the example json. We should go with whichever case is correct.